### ISSUE TITLE
Win 10 CPU 100%, mouse cursor flashes "loading" every second

### ISSUE THREAD

**Issue by TooManyCabinets** [veyon/veyon#69]
I see this exact behavior on all releases of iTALC 3. I can only use iTALC 2.0.2. The same thing is happening with veyon on win 10 Ent. 64 bit. 

After install and after the service starts, mouse cursor changes to "loading" and flashes constantly, every second. CPU is around 100%.

I am using iTALC a lot and hoped veyon would behave like the 2.0.2 version (but without constant crashes). Seems like this  exact issue remains and it prevents me from installing veyon. Hope it can be addressed and solved.

**Comment by tobydox** [comment]
Please try to disable the option "Enable capturing of layered (semi-transparent) windows" in the VNC server settings.

**Comment by tobydox** [comment]
Closing this due to missing feedback. Feel free to comment.

**Comment by NotEnoughCabinets** [comment]
Hello. I am the original poster of this issue. Had to get new username.

I just installed latest version 4.0.4 and I see the same behavior. Mouse switching to "loading" circle near cursor every second when the service is started. When I stop it in Configurator, the cursor looks normal. CPU is not 100% but in Process Explorer I see every few seconds veyon service restarting.

Tried changing VNC server settings as suggested, and some others, did not help. 

I got a Win10 Ent x64 1709. Hardware is HP ProDesk 600 G1.

Is there anything I can try to fix this issue?
